src/Nri/Ui/Select/V4.elm

```diff
@@ -0,0 +1,97 @@
+module Nri.Ui.Select.V4 exposing
+    ( Config
+    , view
+    )
+
+{-| Build a select input.
+
+
+## Upgrading to V3
+
+  - Remove the old styles. V3 uses compile-at-runtime elm-css!
+
+
+# Configure
```

Comment:
🎨 I think this "Configure" section can be combined with the "Render" section, since the `Config` type is only used for calling `view`.